config/routes.rb

```diff
@@ -3,4 +3,8 @@
 
   # Defines the root path route ("/")
   # root "articles#index"
+
+  get "/customers/:id", to: "customers#show"
+  post "/customers/:id", to: "customers#show"
```

Comment:
The more RESTful option here would be 
`post "/customers/:customer_id/items", to: "customer_items#create"`

Remember that from that action you can redirect back to the show page to display the same page with the newly added item!